hikey.mk

```diff
@@ -278,10 +278,10 @@ filelist-all: busybox
 	export TOP=$(ROOT); export MULTIARCH=$(MULTIARCH); \
 	$(expand-env-var) <$(PATCHES_PATH)/rootfs/initramfs-add-files.txt >> $(GEN_ROOTFS_PATH)/filelist-all.txt; \
```

Comment:
It seems that only this line (expand-env-var) needs `TOP` and `MULTIARCH` to be set. So you may as well remove all the line continuation characters from the subsequent lines (well not inside the `if` of course) and keep the `@`. I think it would be a better solution because it's not the first time this kind of mistake happens IIRC.